sonic-xcvrd/scripts/xcvrd

```diff
@@ -95,10 +96,35 @@ platform_chassis = None
 # by DaemonXcvrd
 helper_logger = logger.Logger(SYSLOG_IDENTIFIER)
 
+# The empty namespace refers to linux host namespace.
+EMPTY_NAMESPACE = ''
+
 #
 # Helper functions =============================================================
 #
 
+# Get namespaces, for single ASIC platform namespace is EMPTY_NAMESPACE
+def get_front_end_namesapces():
```

Comment:
Typo: s/get_front_end_namesapces/get_front_end_namespaces/

Also, this function seems like a good candidate function for sonic-py-common. No?

Reply:
These API's are actually wrapper API's which will get the ( for eg: front end namespaces ) irrespective of whether it is a single asic or multi-asic platform. Yes I can definitely move them to multi_asic.py. 

Currently multi_asic.py .. have the API's used by only platforms with >1 ASIC ) -- May we could have some comments in the same file and start adding API's which could be used by both single/multi-asic platforms.

Reply:
I think that makes sense. Do you intend to commit this as-is, then move the functions and remove them from here later, or open a PR to add them to sonic-py-common first, then remove them from this PR?

Reply:
let me open a new PR add these to sonic-py-common first. Later update this PR ... that should be clean.

Reply:
Raised this PR, to add these API's ... https://github.com/Azure/sonic-buildimage/pull/5224